dohclient.go

```diff
@@ -287,6 +287,11 @@ func dohQuicTransport(endpoint string, opt DoHClientOptions) (http.RoundTripper,
 		lAddr = opt.LocalAddr
 	}
 
+	qConf := &quic.Config{}
+	if opt.Use0RTT && opt.Method == http.MethodGet {
```

Comment:
Are you sure about this change? Doesn't it make sense for there to always be a token store, regardless of 0RTT?

Reply:
I am not 100% sure about this. Alternatively `SessionTicketsDisabled = true` could be used but that does basically the same.
I have not found an alternative way. From my testing experience, the use of 0RTT with a custom http.transport seemed very aggressive. 
Ideally I would expect a regular 1RTT opening when using anything but http.MethodGet0RTT but this seems to not be the case.